CHANGELOG.md

```diff
@@ -6,6 +6,18 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
  - Initial Dash 1.0 alpha release [#5](https://github.com/plotly/dash-core/issues/5)
 
+## [0.13.5] - 2019-01-11
+### Changed
+- Added `.idea`, `tests`, `dist`, `.circleci` to npmignore.
+- Added repository url and long_description to setup.py
+
+### Removed
+- Removed click events - these have been obsolete since 0.7.0 [#89](https://github.com/plotly/dash-html-components/pull/89)
+
+## [0.13.4] - 2018-12-17
+### Fixed
+- Fix build from wrong dash version.
+
```

Comment:
Not sure what the good practice would be here but I'd like something to let me know there is something unreleased in this v1.x branch so I can keep track of v1 releases states vs v0 if there's a need for it.

Maybe adding [Unreleased] entry at the top, referring to the 0.13.4 & 0.13.5 version bumps?

Reply:
Oh right I meant to bring up the changelog - it's a little funny since the history isn't quite linear but your idea seems like the most useful approach -> https://github.com/plotly/dash-html-components/pull/90/commits/be67099347009ac38d2b8ea6046828c51a6a9585
To be removed/cleaned up once we have a new rc release.

Reply:
@alexcjohnson Yes exactly. Once in rc, we can collapse the entire alpha history, same thing happens to the rc's once we do an official release.